### PR TITLE
Update nato thread reference

### PR DESCRIPTION
**Pull request by stephen304**
Changes include:

* Removing redundant operations
* Flipping the female thread so it looks upright
* Tweaking the positioning so both threads are flat on the y plane
* Add chamfer to outer edge of the female thread
* Instead of cutting the male thread to shorten it, adjust the chamfer operation

### CODE REVIEW

**Comment by stephen304** [comment]
I'm wondering if it would be good to also add a hole in the male thread with the recommended diameter since not having too thin walls are probably important for the seal on the gasket. So I may add that on to this before merging.

**Comment by stephen304** [comment]
I added a 33mm hole in the male threads since that's what the spec suggests

**Comment by stephen304** [comment]
@scottyallen I think this is ready to merge now if it looks good to you

File: reference/README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
-These are bits and pieces we used in building the mask and accesories!
+# Reference
```
Comment (scottyallen) [inline reference/README.md:1]: These explanations on your design thinking and process are awesome.  Really great work!

**Comment by scottyallen** [comment]
Looks great!

**Pull request merged**